manifests/rhoai/shared/apps/model-catalog/model-catalog-rbac.yaml

```diff
@@ -13,7 +13,7 @@ rules:
       - configmaps
     resourceNames:
       - model-catalog-sources
----
+      - model-catalog-unmanaged-sources
```

Comment:
You removed the `---` separating the Role and RoleBinding here, so this won't work without adding that back